tests/scribeAuth.test.ts

```diff
@@ -2,101 +2,211 @@ import { Auth, Tokens } from '@scribelabsai/auth';
 import { HttpRequest } from '@smithy/protocol-http';
 import * as dotenv from 'dotenv';
 import { describe, expect, it } from 'vitest';
+import { authenticator } from 'otplib';
 
 dotenv.config();
 
 const clientId = process.env['CLIENT_ID']!;
 const username = process.env['USER']!;
+const username2 = process.env['USER2']!;
 const password = process.env['PASSWORD']!;
 const userPoolId = process.env['USER_POOL_ID']!;
 const federatedPoolId = process.env['FEDERATED_POOL_ID']!;
+const otp = process.env['OTPCODE']!;
 const access = new Auth({ clientId, userPoolId });
 const poolAccess = new Auth({
   clientId: clientId,
   userPoolId: userPoolId,
   identityPoolId: federatedPoolId,
 });
 
+const mfaActivatedError = 'challenge instead of tokens';
+
 describe('Get tokens', () => {
   it('Username and password passes', async () => {
     const tokens = await access.getTokens({ username, password });
-    expect(assertTokens(tokens)).toBeTruthy();
+    if ('refreshToken' in tokens && 'idToken' in tokens && 'accessToken' in tokens) {
+      expect(assertTokens(tokens)).toBeTruthy();
+    } else {
+      throw new Error(mfaActivatedError);
```

Comment:
why are we throwing from inside the test?

Reply:
we're returning Tokens | Challenge, so it's a way of not having red underlined vars, plus we'll know if something has changed and is broken instead of skipping

Reply:
Why can't we just `expect(assertTokens(tokens)).toBeTruthy()` without the if nor the else? If it's not a token that will throw

Reply:
![image](https://github.com/ScribeLabsAI/ScribeAuthNode/assets/26488153/5cf766f4-bbaa-4e52-adec-2012b075f878)
that's the underline i was talking about, should i leave that and remove the if else?

Reply:
![image](https://github.com/ScribeLabsAI/ScribeAuthNode/assets/26488153/48168f1c-30a5-4ab2-8345-eb21ffdd7bee)
this is the underline i was talking about, should i leave it and get rid of the if else?

Reply:
well, probably the best is for `assertTokens` to accept `Tokens | Challenge` and fail if it's a `Challenge`